### PR TITLE
Feature/discount badge

### PR DESCRIPTION
**Pull request by vitoriaHeliane**
#### What is the purpose of this pull request?
Add the discount badge component.

#### What problem is this solving?
Add the first version of this component.

#### How should this be manually tested?
<div className="relative dib">
      <DiscountBadge listPrice={200} sellingPrice={20}/>
      
</div>

#### Screenshots or example usage
https://drive.google.com/file/d/1GEpqrRhMiaxqDvEXMJnGMjvlG7_l541u/view?usp=sharing

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/DiscountBadge.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import { FormattedNumber } from 'react-intl'
+import PropTypes from 'prop-types'
+
+/**
+* The discount badge component. It recieves the product's list and selling prices
```
Comment (klzns) [inline react/DiscountBadge.js]: Typo ` recieves` -> `receives`

File: react/DiscountBadge.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    const listPrice = this.props.listPrice
+    const sellingPrice = this.props.sellingPrice
+    return (listPrice - sellingPrice) / listPrice
+  }
+
+  render() {
+    const percent = this.calculateDiscountTax()
+    return (percent !== 0) &&
+      <div className="f7 dark-gray absolute right-0 pa2-s bg-white">
+        <FormattedNumber value={percent} style="percent" /> OFF
```
Comment (klzns) [inline react/DiscountBadge.js]: O texto `OFF` poderia ser uma prop chamada `label` com `defaultValue` sendo `OFF`. O que acha?

Comment (vitoriaHeliane) [reply react/DiscountBadge.js]: Concordo, vou ajustar.

File: react/DiscountBadge.js (hunk 1 above)
Comment (vitoriaHeliane) [reply react/DiscountBadge.js]: Corrigido!

File: react/DiscountBadge.js (hunk 2 above)
Comment (vitoriaHeliane) [reply react/DiscountBadge.js]: Corrigido!

File: react/DiscountBadge.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+/**
+* The discount badge component. It receives the product's list and selling prices
+* and calculates the discount percent to show it in the product's sumary.
+*/
+class DiscountBadge extends Component {
+  /**
+  * This method calculates the discount tax by the selling and list prices.
+  */
+  calculateDiscountTax() {
+    const listPrice = this.props.listPrice
```
Comment (leticiawanderley) [inline react/DiscountBadge.js]: Aqui você pode descontruir os `props`, acho que fica mais legal.
`const {listPrice, sellingPrice} = this.props` 
http://es6-features.org/#ObjectMatchingShorthandNotation

Comment (vitoriaHeliane) [reply react/DiscountBadge.js]: Adicionei isso, realmente fica mais bacana.

**Pull request merged**